terraform/files/bin/apply_nginx_ingress.sh

```diff
@@ -1,8 +1,12 @@
 #!/bin/bash
 export KUBECONFIG=~/.kube/config
 . ~/bin/cccfg.inc
+. ~/$CLUSTER_NAME/harbor-settings
 # Are we enabled? Has a version been set explicitly?
 DEPLOY_NGINX_INGRESS=$(yq eval '.DEPLOY_NGINX_INGRESS' $CCCFG)
+if test "$DEPLOY_NGINX_INGRESS" = "false" -a "$DEPLOY_HARBOR" = "true" -a -n "$HARBOR_DOMAIN_NAME"; then
+  DEPLOY_NGINX_INGRESS="true"
```

Comment:
Consider adding a log message that Harbor forced the nginx-ingress deployment. `echo ...`

Reply:
Addressed in 3b86ce283d77c80d1b6ee3415379ea1035ef7699